package.json

```diff
@@ -51,6 +51,7 @@
     "prettier-plugin-solidity": "^1.0.0-beta.1",
     "pretty-quick": "^2.0.1",
     "secp256k1": "^3.7.1",
+    "solc-0.8": "npm:solc@^0.8.4",
     "solidity-docgen": "^0.5.3",
     "truffle": "^5.2.3",
```

Comment:
Doesn't seem like its needed yet but we should probably upgrade truffle as well, as it looks like there were some network performance features added since this update